recipes/r-modeva/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = '3.17' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-modeva
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/modEvA_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/modEvA/modEvA_{{ version }}.tar.gz
+  sha256: f27ba3da0bbbb61d57e188cbfdf1804d412b8ba256ddfa5fef7da261e5d8f3b0
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-terra >1.5_50
```

Comment:
Actually there is a bit of a problem that this build on windows blocks the others from running. Could you please move this out to a separate submission?

Reply:
Oh, I see, but the others will fail too on Windows since this is a dependency. So non of the recipes here are expected to have successful Windows runs?

Reply:
Done: https://github.com/conda-forge/staged-recipes/pull/26897

I'll update this one when #26897 is sorted out.